### ISSUE TITLE
Dependency conflict between airbrake-JS and node-sass or jsdom/jest

### ISSUE THREAD

**Issue by letorbi** [airbrake/airbrake-js#472]
Hi,

I've recently tried to update the Airbrake-JS package in our project to the latest version, but unfortunately I ran into some problems.

Any `airbrake-js` version >= 1.3.0 causes webpack to fail with the following error:

```
These dependencies were not found:

* fs in ./node_modules/request/lib/har.js
* net in ./node_modules/forever-agent/index.js, ./node_modules/tough-cookie/lib/cookie.js and 1 other
* tls in ./node_modules/forever-agent/index.js, ./node_modules/tunnel-agent/index.js

To install them, you can run: npm install --save fs net tls
```

Installing `fs` `net` & `tls` via yarn or npm solves the problems with `net` and `tls`, but `fs` still cannot be found :thinking: 

Upgrading to `airbrake-js` version 1.2.0 doesn't cause any problems.

A colleague of mine suggested the following explanation for the problems:

> It looks like airbrake-js >= 1.3.0 brings their own request package (included inside airbrake-js/dist/client.js), and this seems to collide with the request package in our build, which is a dependency of i.e. node-sass or jsdom/jest.

Can this really be the reason and is there any chance that  you can fix the problem? Or do you have an idea how to work around the problem, if you won't fix it?

Bye,
Torben

**Comment by vmihailenco** [comment]
airbrake-js imports `request` package in node specific reporter - https://github.com/airbrake/airbrake-js/blob/master/src/reporter/node.ts#L13-L17. Do you see how import like `request = require('request')` could cause such problem?

**Comment by letorbi** [comment]
d1d4db6d8e5289d990915755d521ddaaf6511bf6 seems to have fixed the problem.

**Comment by letorbi** [comment]
@vmihailenco Is there a release date for 1.5.0, yet? I'm asking because I'm not sure, if the product owners like to have a beta-package in their project ;)

**Comment by vmihailenco** [comment]
Just released v1.5.0
